uint8 arm_id
uint8 type_id
float32[] arm_pose